alonet/torch2trt/TRTEngineBuilder.py

```diff
@@ -64,7 +64,7 @@ def __init__(
         self.INT8_allowed = INT8_allowed
         self.onnx_file_path = onnx_file_path
         self.calibrator = calibrator
-        self.max_workspace_size = GiB(8)
+        self.max_workspace_size = GiB(1)
```

Comment:
- In my opnion, this one shouldn't be hardcoded. You can add an argument with 1 as default value.